modules/tensor_mechanics/doc/content/syntax/Modules/TensorMechanics/CohesiveZoneMaster/index.md

```diff
@@ -0,0 +1,36 @@
+# Cohesive Zone Model
+
+## Description
+
+The implemented cohesive zone model (CZM) is based on a Discrete Galerkin approach and therefore does not require cohesive elements. The CZM is formulated in terms of traction separation laws and requires three ingredients:
```

Comment:
Add some introductory text here at the beginning of the paragraph.  Something like this:

The TensorMechanics system provides a cohesive zone modeling capability that can be used to introduce traction-separation models running at the interfaces between regions modeled with continuum finite elements.

Reply:
I included what you suggested